specs/_features/eip7547/fork-choice.md

```diff
@@ -0,0 +1,111 @@
+# EIP-7547 -- Fork Choice
+
+## Table of contents
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Helpers](#helpers)
+  - [`verify_inclusion_list`](#verify_inclusion_list)
+- [Updated fork-choice handlers](#updated-fork-choice-handlers)
+  - [`on_block`](#on_block)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This is the modification of the fork choice accompanying EIP-7547.
+
+## Helpers
+
+### `verify_inclusion_list`
+*[New in EIP-7547]*
+
+```python
+def verify_inclusion_list(state: BeaconState, block: BeaconBlock, inclusion_list: SignedInclusionList,
+                          execution_engine: ExecutionEngine) -> bool:
+    """
+    returns true if the inclusion list is valid. 
+    """
+    # Check that the inclusion list corresponds to the block proposer
+    signed_summary = inclusion_list.signed_summary
+    proposer_index = signed_summary.message.proposer_index
```

Comment:
The proposer index is used here

Reply:
yup modified to not be using the state. put index as part of the IL